Tools/CMake/AMReXParallelBackends.cmake

```
@@ -211,6 +211,7 @@ if (AMReX_HIP)
    find_package(rocprim REQUIRED CONFIG)
    find_package(hiprand REQUIRED CONFIG)
    target_link_libraries(amrex PUBLIC hip::hiprand roc::rocrand roc::rocprim)
+   target_link_libraries(amrex PUBLIC $<$<COMPILE_LANGUAGE:CXX>:${HIP_LIBRARIES}>)
```

Oh right, that does not work...
```suggestion
   target_link_libraries(amrex PUBLIC $<$<LINK_LANGUAGE:CXX>:${HIP_LIBRARIES}>)
```